ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -2518,7 +2518,7 @@ public sealed class Bot : IAsyncDisposable {
 		ArgumentNullException.ThrowIfNull(notification);
 
 		if ((notification.chat_group_id == 0) || (notification.chat_id == 0) || (notification.steamid_sender == 0)) {
-			ArchiLogger.LogNullError($"{nameof(notification.chat_group_id)} || {nameof(notification.chat_id)} || {nameof(notification.steamid_sender)}");
+			ArchiLogger.LogNullError(null, $"{nameof(notification.chat_group_id)} || {nameof(notification.chat_id)} || {nameof(notification.steamid_sender)}");
```

Comment:
We must change it then, I suggest splitting into three `if` checks and three standalone log nulls.